src/gamemode_defense.cpp

```diff
@@ -720,58 +664,58 @@ void defense_game::setup()
 
 void defense_game::refresh_setup( const catacurses::window &w, int selection )
 {
+    // *INDENT-OFF* to preserve column alignment and human readability
```

Comment:
I see this used a few other places to keep "columns" of code aligned. I hope it's acceptable here.